File: programmers/2016년.js
function solution(a, b) {
  // set dates
  let date = ["FRI", "SAT", "SUN", "MON", "TUE", "WED", "THU"]
  // set days in each month
  let days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
  
  let totalDate = days.splice(0, a - 1).reduce((acc, cur) => acc + cur, 0) + b
  
  const idx = (totalDate - 1) % 7;
  
  return date[idx];
}

const a = 5,
    b = 24;

const result = solution(a, b)
console.log(result)

File: programmers/2016년.py
def solution(a, b):
    months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30]
    days = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
    index = 5
    answer = sum(months[:a - 1]) + b
    answer = answer + 4
    answer = answer % 7
    return days[answer]